src/witan/models/dem/ccm/mort/mortality_mvp.clj

```diff
@@ -71,6 +71,37 @@
                                                                    start-yr-avg-mort
                                                                    end-yr-avg-mort)})
 
+(defworkflowfn project-asmr-average-applynationaltrend
+  "Takes a dataset with projected death rates for the future, a dataset with 
+  historic mortality rates, and parameters for: first projection year, last
+  projection year, the scenario to use in the future rates dataset (e.g. :low, 
+  :principal or :high), the start and end  years of data to average for the calculation, 
+  and the last year of mortality data. Returns a dataset that now includes projected
+  mortality rates, projected with the jumpoff year average method combined with an 
+  applied national trend (see docs)"
+  {:witan/name :ccm-mort/project-asmr-average-applynationaltrend
+   :witan/version "1.0"
+   :witan/input-schema {:historic-asmr HistASMRSchema :future-mortality-trend NationalTrendsSchema}
```

Comment:
just for consistency with Ben's doc, please change :future-mortality-trend to :future-mortality-trend-assumption